### ISSUE TITLE
[4.3.4] Registration Form menu item redirection

### ISSUE THREAD

**Issue by joomleb** [joomla/joomla-cms#37566]
Hi guys,
Joomla 4.3.4
while for the **Login process** / **Logout process** we have the very useful settings:

- Login Form menu item > Options > Choose Login Redirect Type: …
- Login Form menu item > Options > Menu Item Login Redirect: …
...
- Login Form menu item > Options > Choose Logout Redirect Type
- Login Form menu item > Options > Menu Item Logout Redirect

For the **Registration process** we have no a redirection setting (there is nothing also in the User Options).
So, at the end of the Registration the User is redirected to the .../registration?layout=complete page, that is simply an empty fixed page with the success message.
Take a look into the …/components/com_users/src/Controller/**RegistrationController.php** file around lines 233 / 242

Would be good to have the same settings as for Login:

- Registration Form menu item > Options > Choose Registration Redirect Type: …
- Registration Form menu item > Options > Menu Item Registration Redirect: …

**Comment by richard67** [comment]
I don’t think we will get this into 3.10 because it would be a new feature, and for the same reason it would not be fixed in 4.1 but in 4.2.

**Comment by joomleb** [comment]
Hi @richard67 
Should it be added to the 4.2 release that is on the way?

**Comment by richard67** [comment]
> Hi @richard67
> Should it be added to the 4.2 release that is on the way?

@joomleb Not my decision. But if you want and can, feel free to make a pull request. Otherwise you to wait until someone else does.

**Comment by joomleb** [comment]
> 
> @joomleb Not my decision. But if you want and can, feel free to make a pull request. Otherwise you to wait until someone else does.

@richard67
sorry for the misunderstanding and thanks for the explication

**Comment by brianteeman** [comment]
I guess this will need to be updated to 4.3

**Comment by joomleb** [comment]
Hi guys,
@HLeithner I noted you are reordering and escalating the tickets for next 4.4 and 5
From my point of view this is a "simple" but "big" missing for Joomla in 2023 and J5 version release...

**Comment by HLeithner** [comment]
@joomleb as you might know joomla 100% volunteer driven, if no one pick the issue to solve it, it might never been solved. So if you are able to contribute the solution or find someone who can help creating a PR we can solve this.

**Comment by joomleb** [comment]
@HLeithner 
Thank You. I'm not a real developer, but I'm going to take a look at it. Please, Can you address me to the
- Login Form menu item
- Registration Form menu item
files directories ?
